endpoint.go

```diff
@@ -500,8 +500,10 @@ func (ep *endpoint) sbJoin(sb *sandbox, options ...EndpointOption) error {
 		return err
 	}
 
-	if e := ep.addToCluster(); e != nil {
-		log.Errorf("Could not update state for endpoint %s into cluster: %v", ep.Name(), e)
+	if sb.ingress {
```

Comment:
can you please explain why is this change needed ?
Infact, it looks like you are trying to add the service to the cluster only for ingress sandbox case ?